encryption/src/codec-server.ts

```diff
@@ -0,0 +1,91 @@
+import express from 'express';
+import cors from 'cors';
+import * as proto from '@temporalio/proto';
+import { EncryptionCodec } from './encryption-codec';
+
+type Payload = proto.temporal.api.common.v1.IPayload;
+
+interface JSONPayload {
+  metadata?: Record<string, string> | null;
+  data?: string | null;
+}
+
+interface Body {
+  payloads: JSONPayload[];
+}
+
+/**
+ * Helper function to converta valid proto JSON to a payload object.
+ *
+ * This method will be part of the SDK when it supports proto JSON serialization.
+ */
+function fromJSON({ metadata, data }: JSONPayload): Payload {
+  return {
+    metadata:
+      metadata &&
+      Object.fromEntries(Object.entries(metadata).map(([k, v]): [string, Uint8Array] => [k, Buffer.from(v, 'base64')])),
+    data: data ? Buffer.from(data, 'base64') : undefined,
+  };
+}
+
+/**
+ * Helper function to convert a payload object to a valid proto JSON.
+ *
+ * This method will be part of the SDK when it supports proto JSON serialization.
+ */
+function toJSON({ metadata, data }: proto.temporal.api.common.v1.IPayload): JSONPayload {
+  return {
+    metadata:
+      metadata &&
+      Object.fromEntries(
+        Object.entries(metadata).map(([k, v]): [string, string] => [k, Buffer.from(v).toString('base64')])
+      ),
+    data: data ? Buffer.from(data).toString('base64') : undefined,
+  };
+}
+
+async function main(port = 8888) {
+  const converter = await EncryptionCodec.create('test-key-id');
```

Comment:
```suggestion
  const codec = await EncryptionCodec.create('test-key-id');
```

Reply:
I think it's best to try and avoid confusion between codec and converter.

Reply:
Agree, I missed this, probably copied from some other file.